CONTRIBUTING.md

```diff
@@ -25,11 +25,16 @@ ZFS-LocalPV is an Apache 2.0 Licensed project and all your commits should be sig
 ## Pull Request Checklist
 * Rebase to the current master branch before submitting your pull request.
 * Commits should be as small as possible. Each commit should follow the checklist below:
-  - For code changes, add tests relevant to the fixed bug or new feature.
+  - For code changes, add tests relevant to the fixed bug or new feature. 
+  - Before commit your code, make sure you have run `make format`.
   - Pass the compile and tests - includes spell checks, formatting, etc.
   - Commit header (first line) should convey what changed and it should follow the commit [guideline](https://github.com/openebs/openebs/blob/master/contribute/git-commit-message.md)
   - Commit body should include details such as why the changes are required and how the proposed changes help
-  - DCO Signed
+  - DCO Signed  
+*  if you have changed crd(comment or anything), then run make manifests to generate the crds as they are auto generated. So you need to do two things:  
```

Comment:
can we remove this line  and append `make manifests` with `make format` in the above line (line 29).

Before committing your code, make sure you have run `make format` and `make manifests`, to format the code and autogenerate the CRDs yaml.